### PR TITLE
ISSUE-242 Remove workarounds for failing CIBA test with bad_binding_message

### PR DESCRIPTION
**Pull request by tnorimat**
closes https://github.com/keycloak/kc-sig-fapi/issues/242

### CODE REVIEW

**Comment by tnorimat** [comment]
@mposolda I've confirmed that all CIBA tests (ping, poll) have been passed as regression. Could you review it? I'm not sure whether I delete the point 9 of "https://github.com/keycloak/kc-sig-fapi/blob/main/README.md#run-fapi-conformance-test-plan-manually-against-the-online-conformance-testsuite" as you've mentioned.

File: README.md [hunk 1]
```diff
@@ -359,10 +335,6 @@ u := "https://192-168-0-101.nip.io:543/auth/realms/test/protocol/openid-connect/
   * After importing the client configurations from the `fapi-conformance-suite-configs` directory, you may need to replace the URLs of
   the keycloak server like using `as.keycloak-fapi.org` with your publicly available Keycloak instance like `https://85-244-72-90.nip.io:543/auth/...`.
   Same applies for the resource server URL, which is `rs.keycloak-fapi.org` in the configuration files by default, and needs to be replaced with your publicly available one.
```
Comment (mposolda) [inline README.md:337]: I would change this line to:
Same applies for the resource server URL, which is `rs.keycloak-fapi.org` and CIBA approval endpoint, which is `aes.keycloak-fapi.org` in the configuration files by default. These needs to be replaced with your publicly available one.

**Comment by mposolda** [comment]
@tnorimat Thanks for this!

I've suggested one minor change in the README docs. Otherwise it looks good to me.

**Review by mposolda — COMMENTED** [review]
Commented above

File: README.md (hunk 1 above)
Comment (tnorimat) [reply README.md]: I've fixed.

**Comment by tnorimat** [comment]
@mposolda Could you check the point you've pointed out? If OK, could you merge this PR?

**Comment by mposolda** [comment]
@tnorimat Thanks!

**Pull request merged**